Some test cases in IoUtilTest class fails on Macos

### Describe the bug

When running tests in `org.eclipse.jkube.kit.common.util.IoUtilTest`, some test cases fail with a message similar to the following one:

```
java.lang.AssertionError: 
Expecting actual:
  ["../../../../../../private/var/folders/5q/fb9_qmc91h76mqsnyqxnf7x00000gn/T/junit14779572735641380464/deployment.yaml"]
to contain exactly in any order:
  ["deployment.yaml"]
elements not found:
  ["deployment.yaml"]
and elements not expected:
  ["../../../../../../private/var/folders/5q/fb9_qmc91h76mqsnyqxnf7x00000gn/T/junit14779572735641380464/deployment.yaml"]
```
After reading the implementation of method `fileTree` in class `FileAssertions`, I found out that the relative path of files in the file tree is not computed correctly on MacOS.

P.S. Fixed the issue. Shall I create a PR for it?

### Eclipse JKube version

SNAPSHOT

### Component

JKube Kit

### Apache Maven version

None

### Gradle version

None

### Steps to reproduce

1. It requires MacOS
2. Run `mvn clean test` inside `jkube-jit` module
3. Observe the error

### Expected behavior

Tests inside class `IoUtilTest` should all pass.

### Runtime

other (please specify in additional context)

### Kubernetes API Server version

1.25.3

### Environment

macOS

### Eclipse JKube Logs

_No response_

### Sample Reproducer Project

_No response_

### Additional context

_No response_

--- a/jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/assertj/FileAssertions.java
+++ b/jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/assertj/FileAssertions.java
@@ -42,7 +42,7 @@ public class FileAssertions extends AbstractFileAssert<FileAssertions> {
   public AbstractListAssert<ListAssert<String>, List<? extends String>, String, ObjectAssert<String>> fileTree()
       throws IOException {
 
-    final Path actualPath = actual.toPath().normalize();
+    final Path actualPath = actual.toPath().normalize().toRealPath();
     final List<String> paths = new ArrayList<>();
     try (Stream<Path> pathStream = Files.walk(actualPath)) {
       for (Path temp : pathStream

